app/controllers/rb_application_controller.rb

```diff
@@ -39,6 +39,8 @@ class RbApplicationController < ApplicationController
 
   before_filter :load_sprint_and_project, :check_if_plugin_is_configured, :authorize
 
+  skip_before_action :verify_authenticity_token, if: -> { Rails.env.test? }
```

Comment:
@ulferts I don't much like this either, but this workaround exists for the benefit of `features/step_definitions/_when_steps.rb` where, for some reason (I imagine previous developers were having issues properly simulating drag-drop with Capybara), we instruct the browser to make direct HTTP requests - e.g. `page.driver.post`